include/openssl/bn.h

```diff
@@ -1066,7 +1071,6 @@ OPENSSL_EXPORT unsigned BN_num_bits_word(BN_ULONG l);
 #define BN_FLG_STATIC_DATA 0x02
 
 #ifdef AWS_LC_INTERNAL_IGNORE_BN_SET_FLAGS
```

Comment:
We should also delete this #ifdef as we can just add a -DBN_FLG_CONSTTIME=0x04 at build time OR properly #ifdef the upstream packages (OpenSSH comes to mind).

Reply:
Agree 100%